### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by Alusp**
### In this PR

-  I added an Email upper case validation and error message

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -136,8 +162,32 @@ function projectPopup(position) {
...
+// ######################## Contact Form Validation Section #######################
+const validateEmail = (email) => {
+  if (email !== email.toLowerCase()) return false;
+  return true;
+};
+
+contactForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (!validateEmail(contactForm.elements.email.value.trim())) {
+    alert('Email should be all lower case only');
```
Comment (1GORDON) [inline script.js:189]: Great job team 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md), you are supposed to print the error message next to the submit button. This is not the case for your design. You may try not to use the alert function to render the error message.
alert

File: style.css [hunk 2]
```diff
@@ -810,7 +764,7 @@ ul {
   width: 327px;
   height: 170px;
   border-radius: 8px;
-  margin-top: 10px;
+  padding: 10px;
 }
 
 .submission {
```
Comment (1GORDON) [inline style.css:770]: Great job team 🤝

- [ ] Kindly fix the overlapping submit button on your mobile design as it deviates from the design provided in the Figma template.
- Your design
overlap mobile

- Figma design
Figma design

File: style.css [hunk 3]
```diff
@@ -842,11 +796,12 @@ ul {
...
 }
 
-.submission :hover:enabled {
+/* .submission :hover:enabled {
   background: #6070ff;
   color: white;
   box-shadow: calc(0px + 0px) 8px 16px rgb(64 83 252 / 24%);
-}
+  border-color: #c1c7d0;
+} */
```
Comment (1GORDON) [inline style.css:804]: Great work team 🤝

- [ ] Kindly remove all commented code from your style sheet as this is not regarded as a good practice.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Alusp & @maaloum ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull Request ✔️
- Professional Readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**